src/core/aws-server.ads

```diff
@@ -235,13 +235,14 @@ package AWS.Server is
       Host          : String;
       Port          : Natural;
       Family        : Net.Family_Type := Net.Family_Unspec;
-      Reuse_Address : Boolean         := False)
-   with Pre => Host'Length > 0;
```

Comment:
This removal was probably not intended.

Reply:
I removed the "Pre => Host'Length > 0;" because empty Host meaning is bind to all local addresses. When we support IPv6 the secondary bind can be to all addresses too, but for different address family.

Reply:
I see!